src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -149,9 +149,13 @@ ldap but it felt unnatural.
     roles: Set[ResourceRoleName],
     actions: Set[ResourceAction],
     public: Boolean)
-    extends WorkbenchGroup
+    extends WorkbenchGroup {
+  def metadata: AccessPolicyMetadata = AccessPolicyMetadata(id, email, roles, actions, public)
+}
 @Lenses final case class AccessPolicyMembership(memberEmails: Set[WorkbenchEmail], actions: Set[ResourceAction], roles: Set[ResourceRoleName])
 @Lenses final case class AccessPolicyResponseEntry(policyName: AccessPolicyName, policy: AccessPolicyMembership, email: WorkbenchEmail)
+// Access Policy with no membership info
```

Comment:
nit: Maybe say why this is useful to have? does it model the sql table more closely, do clients care less?